### ISSUE TITLE
Decide on the how of publishing packages

### ISSUE THREAD

**Issue by kasperisager** [Siteimprove/alfa#212]
As Alfa is managed using Yarn and Lerna, I envision the release workflow to be as follows:

1. Once a new release is ready to be published, `lerna version [major | minor | patch]` is run locally which will bump the required package versions and push a `vX.Y.Z` tag to git.

2. An Actions workflow picks up on `v*` tags and kicks off the release process:

    1. Build Alfa from scratch using `yarn install` and `yarn build`.

    2. Publish the packages using `lerna publish from-package`. This ensures that Lerna doesn't attempt to run `lerna version` a second time behind the scenes but instead publishes all packages whose present version doesn't yet exist in the registry.

    3. Create a [GitHub Release](https://help.github.com/en/github/administering-a-repository/about-releases) named according to the tag.

The big question then becomes: To which registry should the packages be published? As Alfa is an open repository, we do have access to [GitHub Packages](https://github.com/features/packages) so that's an obvious candidate if we want to avoid having to set up and manage an npm organisation, which would first need an application and business owner and go through a security review.

**Comment by aclemmensen** [comment]
This might miss the mark somewhat, but would it make sense to have a github organization for alfa specifically? Depending on how you plan to manage contributors etc in the future, it might be nice to have everything isolated instead of having to add individual collaborators to this project.

Other than that, GH packages sounds like a good idea, in particular if it's free for public repos. I believe Jan is considering which GH plan we might switch to which may include the Packages feature as well (see the discussion about distributing npm packages from the other day on Slack).

**Comment by kasperisager** [comment]
It would sure make sense to move to a separate GitHub organisation as we've also discussed previously. Back then, the reason was a wish to use external integrations that needed access to public _and_ private repositories, which was not possible to grant. With the introduction of GitHub Actions, we no longer had such a need and I therefore didn't take the idea any further. Now, it might again make sense to consider moving to a separate GitHub organisation, also, as you mention, to ease the on-boarding of external contributors without risking exposure of our private repositories.

The obvious choice would of course be https://github.com/alfa, but that name is already taken. The user does however not seem to be active so I've sent an email to the address listed and asked if they'd be willing to donate the username.

**Comment by kasperisager** [comment]
In order to avoid the added burden of managing another GitHub organisation and, as some have pointed out, the possible brand-disassociation that might follow I don't see us moving to a separate GitHub organisation just yet. The question of how to manage external contributors will be answered when it hopefully becomes relevant; the simple answer would of course be that the @Siteimprove/alfa team is strictly for Siteimprove employees and that external contributors are required to operate on forks.

Unless there are any further concerns about publishing Alfa to GitHub Packages, which will be free as Alfa is a public repository of open-source code, that's the route we'll take.
